packages/visualizations/addon/components/navi-visualizations/table.js

```diff
@@ -37,7 +37,7 @@ export default Component.extend({
   /**
    * @property {Array} classNames - list of component class names
    */
-  classNames: [ 'table-widget' ],
+  classNames: ["table-widget"],
```

Comment:
We typically use single quotes in JS files. Can we revert the quote changes?

Reply:
weird. I did not make such changes.

Reply:
It is caused by a editor plugin I installed recently